### DISCUSSION TITLE
Does dnycrypt-proxy work system-wide on Windows?

### DISCUSSION THREAD

**Discussion by Skrell** [DNSCrypt/dnscrypt-proxy#2551] [Q&A] [answered]
If an application, like Firefox, uses its own hardcoded dns, will dnscrypt-proxy still intercept the url lookup?

**Comment by lifenjoiner** [comment, marked as answer]
Definitely no.
That communicates with the hard-coded DNS directly.

**Comment by nokia8801** [comment]
It does work system wide in Windows, I used it like that for years. But with Firefox you'll have to disable DoH in the settings before the system wide dnscrypt-proxy is used.

**Reply by ghost** [reply]
> But with Firefox you'll have to disable DoH in the settings before the system wide dnscrypt-proxy is used.

Not necessary, DNSCrypt-Proxy prevents Firefox using DoH unless a user intentionally opts in.

**Reply by nokia8801** [reply]
Oh right, I forgot the Firefox workaround that gets mentioned every time the service starts.
